runtime/syntax/nftables.yaml

```diff
@@ -0,0 +1,19 @@
+filetype: nftables
+
+detect:
+    filename: "nftables.conf$"
+    header: "^#.*?/etc/nftables"
+    
+rules:
+    - statement: "(chain|define|table)"
+    - symbol: "(=)"
+    - diff-modified: "(flush|type|counter)"
+    - special: "^\\[(Unit|Install|Service|Socket|Timer)\\]"
+    - constant.string.char: "(;)"
+    - constant.string: '([\"]{1})(.*)([\"]{1})'
+    - type: "(\\#\\!)"
+    - preproc: "((([$])[a-zA-Z0-9\\_\\-\\+]{1,250})|(([A-Z]{1,})[A-Z\\_]{1,250}))"
+    - constant.bool: "\\b(true|false)\\b"
```

Comment:
Boolean values can only be specified as `1`, `0`, `exists` and `missing`. I do not usually see the last 2 so I do not know  how they would be highlighted.

Reply:
I have removed the lines